committeeoversightapp/management/commands/load_committeeratings.py

```diff
@@ -36,24 +36,27 @@ def build_committee_rating(self, congress, committee):
         # + Nominations + Fact Finding + Field + Closed
         total_hearings=self.count_by_category(
             committee_hearings,
-            ['Nominations', 'Legislative', 'Policy' 'Agency Conduct',
-             'Private Sector Oversight' 'Fact Finding', 'Field', 'Closed']
+            ['Nominations', 'Legislative', 'Policy', 'Agency Conduct', \
```

Comment:
I'm pretty sure you don't need the slash when you're inside of a list definition.